template.rc

```diff
@@ -0,0 +1,57 @@
+/* This is a template RC file.
+ * Do not edit with MSVC */
+#ifdef APSTUDIO_INVOKED
+# error dont edit with MSVC
+#endif
+
+#include "winres.h"
+#include "main/php_version.h"
+#include "phongo_version.h"
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+#pragma code_page(1252)
+
+#ifndef THANKS_GUYS
+# define THANKS_GUYS ""
+#endif
+
+//Version
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION PHP_MONGODB_VERSION_DESC
+ PRODUCTVERSION PHP_MONGODB_VERSION_DESC
+ FILEFLAGSMASK 0x3fL
+#ifdef _DEBUG
+ FILEFLAGS VS_FF_DEBUG
+#else
+ FILEFLAGS 0x0L
+#endif
+ FILEOS VOS__WINDOWS32
+ FILETYPE VFT_DLL
+ FILESUBTYPE VFT2_UNKNOWN
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "Comments", THANKS_GUYS "\0"
+            VALUE "CompanyName", "MongoDB, Inc.\0"
+            VALUE "FileDescription", FILE_DESCRIPTION "\0"
+            VALUE "FileVersion", PHP_MONGODB_VERSION
+            VALUE "InternalName", FILE_NAME "\0"
+            VALUE "LegalCopyright", "Copyright � 2014-2018 MongoDB, Inc.\0"
```

Comment:
Why do I see a Unicode replacement character here?

I'd also suggest using "present" as the upper bound, per [DRIVERS-329](https://jira.mongodb.org/browse/DRIVERS-329). This could be rewritten as:

> Copyright 2014-present MongoDB, Inc.

Reply:
The file encoding is win-1252 (see line 12), I guess GitHub doesn't get that.

Reply:
Ok. I think we should still use "present" as the date's upper bound, though. That will ensure this never goes out of date.